docs/shared/react-tutorial/3-task-running.md

```diff
@@ -0,0 +1,179 @@
+These targets are sufficient for your `products` project, but if more targets were required in the future, you could create more targets by defining them here.
+
+{% callout type="note" title="Configuration-less Tasks"%}
+In addition to configuring your tasks via Nx executors in a `project.json` file, Nx also supports configuration-less setup, where Nx can run tasks as defined by the `scripts` in your project's `package.json` file.
+
+You can read more about configuring your project in [the Project Configuration reference page](/reference/project-configuration).
+{% /callout %}
+
+You can also check the `project.json` files of your other projects in the workspace (`admin`, `admin-e2e`, `store`, `store-e2e`, and `common-ui`) to see the targets already available for them.
+
+## Syntax for Running Tasks in Nx
+
+To run a target in Nx, use the following syntax:
+
+![Syntax for Running Tasks in Nx](/shared/react-tutorial/run-target-syntax.png)
+
+To see this in action, run the `test` target for our `products` project by running the command `npx nx test products`:
+
+```bash
+> npx nx test products
+
+> nx run products:test
+
+FAIL products libs/products/src/lib/products.spec.ts
+● Test suite failed to run
+
+    libs/products/src/lib/products.spec.ts:1:10 - error TS2724: '"./products"' has no exported member named 'products'. Did you mean 'Product'?
+
+    1 import { products } from './products';
+               ~~~~~~~~
+
+Test Suites: 1 failed, 1 total
+Tests: 0 total
+Snapshots: 0 total
+Time: 0.819 s
+Ran all test suites.
+
+———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+
+> NX Ran target test for project products (2s)
+
+    ✖    1/1 failed
+    ✔    0/1 succeeded [0 read from cache]
+
+```
+
+As we can see, our tests are currently failing as we haven't adjusted any of our tests since generating and making some changes to this project.
+
+You can find the `jest.config.ts` file for this project here:
+
+```ts {% fileName="libs/products/jest.config.ts" %}
+/* eslint-disable */
+export default {
+  displayName: 'products',
+  preset: '../../jest.preset.js',
+  globals: {
+    'ts-jest': {
+      tsconfig: '<rootDir>/tsconfig.spec.json',
+    },
+  },
+  transform: {
+    '^.+\\.[tj]s$': 'ts-jest',
+  },
+  moduleFileExtensions: ['ts', 'js', 'html'],
+  coverageDirectory: '../../coverage/libs/products',
+};
+```
+
+The generators you ran in part 1 of this tutorial created this file. Generally, you wouldn't need to touch this file as you add tests, you'd just need to add `<test name>.spec.ts` files anywhere in the project's source directory. You can however adjust the contents of this file (like you would outside the context of an Nx workspace) to match your workspace's specific requirements.
```

Comment:
I ended up removing this part, as I chose to instead use the `test` and `lint` target of `common-ui` project to demonstrate task-running here (since they both pass without further adjustments)